### PR TITLE
Add option set marshmallow on fire for greek story version

### PR DESCRIPTION
**Pull request by mpaktiti**
Please review this, @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2247)

### CODE REVIEW

File: greek/offer_marshmallow_a_cigar/offer_marshmallow_a_cigar.md [hunk 1]
```diff
@@ -1,4 +1,10 @@
...
 δυνατά και του λες περιπαικτικά ότι ακόμη και σ'ένα δωμάτιο από ζαχαρωτά,
-υπάρχει ένας αγενής Έλληνας καπνιστής.'
\ No newline at end of file
+υπάρχει ένας αγενής Έλληνας καπνιστής.'
+
+Αφου καπνίσεις το τσιγάρο που το ζαχαρωτό χέρι αρνήθηκε σκέφτεσαι
+πως είναι ώρα να βγεις από εκεί. Πόρτα δεν υπάρχει οπότε πρέπει
+να χρησιμοποιήσεις κάτι από αυτά που έχεις. Τι κάνεις;
+
+[Ανάβεις τον αναπτήρα](set_it_on_fire/set-it-on-fire.md)
```
Comment (casey-collab) [inline greek/offer_marshmallow_a_cigar/offer_marshmallow_a_cigar.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: greek/offer_marshmallow_a_cigar/set_it_on_fire/set_it_on_fire.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Ανάβεις τον αναπτήρα, ψήνεις το χέρι και το τρως.
```
Comment (casey-collab) [inline greek/offer_marshmallow_a_cigar/set_it_on_fire/set_it_on_fire.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by mpaktiti** [comment]
I fixed the typo (I also introduced a new one and fixed again). Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :thumbsup:

**Pull request merged**